api/routers/export.py

```diff
@@ -192,20 +86,17 @@ async def export(export_info: ExportDTO, background_tasks: BackgroundTasks):
     Returns a zip file containing the CSV files for the requested data.
 
     The endpoint allows filtering by:
-    - *Entity*: (areas or cameras).
+    - *Entity*: (cameras).
     - *Dates*: (only include data for the specified date range).
     - *Data Type*: (the type of information that you want to export. The available values are raw_data, occupancy,
         social-distancing, facemask-usage, in-out and all_data)
     """
-    areas = get_areas_to_export(export_info)
-    cameras = get_cameras_to_export(export_info, areas)
+    cameras = get_cameras_to_export(export_info)
```

Comment:
will the processors support exporting data from cameras? I don't mind from the API but it might be weird from the frontend (unless the only option is all cameras)